pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -64,19 +64,19 @@ public class AggregateOperator extends MultiStageOperator {
       new CountAggregationFunction(Collections.singletonList(ExpressionContext.forIdentifier("*")), false);
   private static final ExpressionContext PLACEHOLDER_IDENTIFIER = ExpressionContext.forIdentifier("__PLACEHOLDER__");
 
-  private final MultiStageOperator _inputOperator;
+  private final MultiStageOperator _upstreamOperator;
```

Comment:
This is confusing. In the mailbox, upstream is the next operator (e.g. receiver is upstream for sender), but here upstream is input operator. Suggest reverting this change because input is more clear